test/inductor/test_mkldnn_pattern_matcher.py

```diff
@@ -3112,6 +3112,70 @@ def matcher_check_fn():
             compile_options={"dynamic": dynamic},
         )
 
+    @skipIfNoDynamoSupport
+    @skipIfNoONEDNN
+    # TODO: investigate options of torch.compile in fbcode
+    @unittest.skipIf(IS_FBCODE, "Failing in fbcode")
+    @parametrize("has_bias", [True, False])
+    @parametrize("dtype", [torch.float, torch.bfloat16])
+    @parametrize("dynamic", [True, False])
+    def test_da8w8_sym_act_sym_wgt_with_int_mm(self, has_bias, dtype, dynamic):
+        r"""
+        This testcase check if we can match the int8_dynamic_activation_int8_weight int8 linear pattern from torchao,
+        when activation is symmetrically quantized dynamically & weights are symmetrically quantized (statically)
+        The pattern is:
+            (no bias) _int_mm -> convert_element_type -> (expand_a) -> mul -> mul
```

Comment:
Should `expand_a` connect to another input edge of `mul`?

Reply:
Yes, this visual depiction just means that the activation scale may be expanded before the `mul` corresponding to it.
The base PR uses the same notation, so I reused it. The UTs have been copy-pasted & modified from the base PR.

Reply:
I added a comment to clarify

Reply:
Thanks, I fixed it. I guess you were referring to including `expand_a -> mul` within brackets.